### DISCUSSION TITLE
Enhance Privacy Features: Integrate WindowsSpyBlocker

### DISCUSSION THREAD

**Discussion by win-open** [memstechtips/UnattendedWinstall#82] [Ideas]
Hello @memstechtips @devCKVargas  @zekurio ,

I recently came across the [WindowsSpyBlocker](https://github.com/crazy-max/WindowsSpyBlocker) program, which offers a robust solution for blocking Windows spyware and telemetry on firewall level. I believe integrating this program or implementing similar functionality in your unattended file could significantly enhance the privacy features of your project.

This is just a proposal as I dont know the technicalities behind the project that much. I think it would be more privacy or security improvement on changing the firewall rules and the NCIS methods by adding those commands in your script (unattended file) or include the program like CTT's utility on the desktop.

Thank you for considering my proposal, and I look forward to your response.

Best regards,
Win

**Comment by memstechtips** [comment]
Hi there, 

thanks for this. Will have a look at it sometime.

**Comment by devCKVargas** [comment]
I use [shutup10++](https://www.oo-software.com/en/shutup10) for this. It's nice that CTT implemented this on his [WinUtil](https://github.com/ChrisTitusTech/winutil/blob/main/docs/userguide.md#oo-shutup).

I need some features, so I don't just blindly enable all recommendations, but the normal privacy concerned windows users won't really care or bother if we implement it in the autounattent.xml or use this https://github.com/ChrisTitusTech/winutil/issues/1921

**Comment by NovaXprss532** [comment]
With Win10privacy (https://www.w10privacy.de/english-home/) you can configure telemetry and firewall rules yourself, as well as many other options to fine-tune your OS. This tool also works with Windows 11
